omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -238,6 +238,7 @@
 # See default['private_chef']['postgresql']['db_connection_superuser'] for information
 default['private_chef']['opscode-erchef']['sql_connection_user'] = nil
 default['private_chef']['opscode-erchef']['enable_request_logging'] = true
+default['private_chef']['opscode-erchef']['allow_email_update_only_from_manage'] = false
```

Comment:
The behavior of this attribute should also be tested in the pedant specs if possible.

Reply:
I tried looking for possibilities but no luck, There is no way to change the existing components configurations via pedant tests.